crates/noirc_frontend/src/monomorphization/printer.rs

```diff
@@ -96,6 +96,11 @@ impl AstPrinter {
             super::ast::Literal::Integer(x, _) => x.fmt(f),
             super::ast::Literal::Bool(x) => x.fmt(f),
             super::ast::Literal::Str(s) => s.fmt(f),
+            super::ast::Literal::FmtStr(_, _, fields) => {
+                write!(f, "fmtstr<")?;
+                self.print_expr(fields, f)?;
+                write!(f, ">")
```

Comment:
We should be printing `N` too here

Reply:
The beginning output of printing the monomorphized AST from the `debug_logs` test:
```
fn main$f0(x$l0: Field, y$l1: Field) -> () {
    let string$2 = i: {i}, j: {j};
    println$f1(string$l2);
    let fmt_str$3 = f"i: {x}, j: {y}";
    let fmt_str$4 = string_identity$f2(fmt_str$l3);
    println$f3(fmt_str$l4);
    let fmt_str_no_type$5 = f"i: {x}, j: {y}";
```